Board.cpp

```diff
@@ -36,51 +36,92 @@ void Board::init()
 	xside = black;
 	hply = 0;
 	castleRights = 15;
+	fifty = 0;
 }
 
-bool Board::checkMove(Move m) const
+Move Board::getMove(int m) const
```

Comment:
can you add some comments here for my sake?

Reply:
Comments about the function are in the header file.